cico_functions.sh

```diff
@@ -44,6 +68,15 @@ function install_deps() {
     git
 
   service docker start
+  
+  #set buildx env
+  export DOCKER_BUILD_KIT=1
+  export DOCKER_CLI_EXPERIMENTAL=enabled
```

Comment:
I'm a bit iffy about relying on experimental features from Docker but if this is the One True Way to do multiarch builds, then so be it.

Reply:
The `buildah` discussion appears to be here: https://github.com/containers/buildah/issues/1590